package.json

```diff
@@ -106,31 +107,23 @@
         "moment": "2.19.3",
         "moment-timezone": "^0.5.4",
         "mousetrap": "^1.6.1",
-        "node-uuid": "^1.4.7",
-        "path-to-regexp": "^1.7.0",
         "prop-types": "^15.7.2",
-        "r-dom": "^2.3.2",
-        "react": "^16.8.6",
-        "react-addons-perf": "15.4.2",
+        "react": "^16.13.1",
```

Comment:
I think we'll need a corresponding component library PR to update to the same React version. IIRC, we ran into issues when running two different versions?

Reply:
I think and hopefully that is only the latest major version change, this is what our app is running on currently at the time of this PR. The `^` allows for increases in minor versions, so we do not even really need this. I just looked our package lock with the new `qat -> staging` PR and it is at `6.14`
<img width="265" alt="Screen Shot 2021-06-16 at 5 21 18 PM" src="https://user-images.githubusercontent.com/14301977/122295693-8596d300-cec7-11eb-9b93-f2102c38d7bf.png">